_data/ols-4-schedule.yaml

```diff
@@ -221,7 +221,6 @@ weeks:
         - Create a GitHub repository
```

Comment:
Can you remove the changes in this file?

Reply:
I have reverted the changes I made earlier.

https://github.com/Chimatara/open-life-science.github.io/blob/Revert-Changes/_data/ols-4-schedule.yaml